demos/general/demo_pipeline_voltage_imaging.py

```diff
@@ -233,34 +234,34 @@ def main():
 
     opts.change_params(params_dict=opts_dict);          
 
-#%% TRACE DENOISING AND SPIKE DETECTION
+    #%% TRACE DENOISING AND SPIKE DETECTION
     vpy = VOLPY(n_processes=n_processes, dview=dview, params=opts)
-    vpy.fit(n_processes=n_processes, dview=dview)
+    vpy.fit(n_processes=n_processes, dview=dview);
```

Comment:
Is this semicolon just there for cell-based evaluation? Does it work that way? I thought that was just a jupyter thing